go.mod

```
@@ -2,7 +2,14 @@ module github.com/rook/rook
 
 go 1.21
 
-replace github.com/rook/rook/pkg/apis => ./pkg/apis
+replace (
+	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
+	github.com/kubernetes-incubator/external-storage => github.com/libopenstorage/external-storage v0.20.4-openstorage-rc3
+	// github.com/libopenstorage/secrets => github.com/sp98/secrets v0.0.0-20240220060802-e2bb123d06b9
+	github.com/libopenstorage/secrets => github.com/sp98/secrets v0.0.0-20240304064404-1ca2f1c12a8b
+	github.com/portworx/sched-ops => github.com/portworx/sched-ops v0.20.4-openstorage-rc3
```

```suggestion
	github.com/portworx/sched-ops => github.com/portworx/sched-ops v0.20.4-openstorage-rc3
```
any reason for this change? As we have put the `portworx` import in  exclude section.

---

thanks for explaining the uses.

---

Thanks for explaining. I marked this it as unresolved to make sure that we keep the conversation more easily visible in case we have to reference it in the future. I don't see a way to mark it as resolved but still see the convo.